### ISSUE TITLE
TypeScript errors for victory-native components

### ISSUE THREAD

**Issue by scottrippey** [FormidableLabs/victory#2451]
The following errors are being reported:

```
node_modules/victory-brush-container/lib/brush-helpers.d.ts(5,41): error TS2694: Namespace '_' has no exported member 'DebouncedFunc'.
node_modules/victory-cursor-container/lib/cursor-helpers.d.ts(7,35): error TS2694: Namespace '_' has no exported member 'DebouncedFunc'.
node_modules/victory-native/src/components/victory-brush-container.d.ts(7,50): error TS2749: 'VictoryBrushContainer' refers to a value, but is being used as a type here. Did you mean 'typeof VictoryBrushContainer'?
node_modules/victory-native/src/components/victory-cursor-container.d.ts(7,51): error TS2749: 'VictoryCursorContainer' refers to a value, but is being used as a type here. Did you mean 'typeof VictoryCursorContainer'?
```

_Originally posted by @adamlutz in https://github.com/FormidableLabs/victory/issues/2358#issuecomment-1230740679_

**Comment by scottrippey** [comment]
@adamlutz I am currently fixing the `'VictoryBrushContainer' refers to a value....` errors.  Thanks for pointing these out!

**Comment by scottrippey** [comment]
@adamlutz Regarding the `Namespace '_' has no exported member 'DebouncedFunc'` ...

I think you should install the`@types/lodash` package to fix this.

Normally, TS does not complain about the `node_modules` folder types, and shouldn't complain about missing Lodash types.  However, if your `tsconfig` has `skipLibCheck: false`, then it will perform a typecheck for all `node_modules` ... and since the `@types/lodash` is a devDependency, it will complain.
So, the easiest workaround here is to install `@types/lodash` yourself, and the error should be resolved.

If this works, please report back here.  Thanks!

**Comment by scottrippey** [comment]
@adamlutz Can you please check the latest version, `36.6.6` to see if those last 2 errors are fixed? (`VictoryBrushContainer` and `VictoryCursorContainer` `refers to a value...`)

**Comment by adamlutz** [comment]
hey @scottrippey confirmed that the last two errors are solved in `36.6.6`.

I did actually already have `@types/lodash` installed, however it seems that rolling the patch version forward seems to have fixed the lodash namespace issues related to victory-native. 

Thanks for your help on this!  Really appreciate the quick responses and quick action on this 🙇
